src/components/GlobalHelper/GlobalHelper.st.css

```diff
@@ -0,0 +1,90 @@
+:import {
+  -st-from: "wix-ui-core/dist/src/baseComponents/Popover/Popover.st.css";
+  -st-default: Popover;
+}
+
+:import {
+  -st-from: "../../palette.st.css";
+  -st-named: brandSecondary, backgroundSecondary
+}
+
+:import {
+  -st-from: "../../typography.st.css";
+  -st-named: fontRoman;
+}
+
+.root {
+  -st-states: bounce, placement-right, placement-left, placement-top, placement-bottom;
+  /* -st-extends: Tooltip; */
+  -st-extends: Popover;
+  white-space: nowrap;
+}
+
+/* popoverContent and innerContent are taken from Tooltip.tooltipContent,
+ * but I split them.
+*/
+.root::popoverContent {
+  border-radius: 8px;
+  padding: 12px 24px;
+  box-sizing: border-box;
+  box-shadow:
+  0 6px 6px 0 rgba(22, 45, 60, 0.06),
```

Comment:
colors should probably be part of the palette

Reply:
this is how it is in the zeplin spec. I'll ask them.